site/content/ytt/docs/latest/lang-ref-ytt-schema.md

```diff
@@ -240,8 +240,21 @@ Explicitly configures the type of the annotated node. Currently, the only suppor
 ```
 
 where:
-- `any` (`bool`) — whether or not any and all types are permitted on this node (and its children).
+- `any` (`bool`) — whether or not any and all types are permitted on this node and its children.
 
+The annotated node and its nested children are not checked by schema, and has no schema default behavior. 
+However, the annotated node and its children are simply passed-through as a data value. 
+All nested `@schema` annotations are ignored.
```

Comment:
I know this has already been merged, but - 
in the context of `@schema/desc ""`, do we want to ignore this annotation if it is nested under a `@schema/type ay=True`? It seems like we could still add this metadata.

Reply:
@jtigger @gcheadle-vmware

Reply:
I can absolutely see this on _the_ node being marked `@schema/type any=True`... but I can't see it for the _contained_ YAML, myself. I believe we're only including it as a default value, but the structure is omitted... so no place to put a `description` property in the output......

or am I missing something?